### PR TITLE
Release 1.0.1

### PR DESCRIPTION
**Pull request by tscurtu**
Release charts version 1.0.1 with Sematic server version 0.22.0.

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+    created: "2023-01-18T01:59:44.866064746Z"
+    description: Sematic AI Server
+    digest: 982e4ef68e13ea72eec1546a2e3ccc526b427a046657e5ddcfe9d4036220a705
+    maintainers:
+    - name: sematic-ai
+      url: https://github.com/sematic-ai/sematic/
+    name: sematic-server
+    type: application
+    urls:
+    - https://sematic-ai.github.io/helm-charts/sematic-server-1.0.1.tgz
```
Comment (neutralino1) [inline index.yaml:15]: Why is the helm chart version different from the release version?

Comment (tscurtu) [reply index.yaml:15]: This is the version of the helm charts, which may evolve differently compared to the server version given work being done on the charts themselves.

File: index.yaml [hunk 2]
```diff
@@ -14,4 +27,4 @@ entries:
     urls:
     - https://sematic-ai.github.io/helm-charts/sematic-server/sematic-server-1.0.0.tgz
     version: 1.0.0
-generated: "2023-01-07T21:41:53.247214-08:00"
```
Comment (snoshy) [inline index.yaml:17]: undo this hunk, we don't want to update timestamps of older releases.  i'll discuss with you tomorrow about why this happened.

**Review by snoshy — CHANGES_REQUESTED** [review]

File: index.yaml (hunk 2 above)
Comment (snoshy) [reply index.yaml:17]: nvm i'm wrong, please ignore

**Review by snoshy — APPROVED** [review]

**Pull request merged**